src/Components/Comment/CommentCard.jsx

```diff
@@ -0,0 +1,31 @@
+import { Link } from "react-router-dom";
+import VoteComponent from "../Vote";
+
+const CommentCard = ({ comment }) => {
+  return (
+    <section className="max-w-3xl mx-auto bg-white rounded-lg shadow-lg p-6 my-6 transition-transform transform hover:scale-105">
```

Comment:
As mentioned above, I'm not sure what you want to achieve with the hover transform. While it's great you've got on with tailwind and are having some lovely results, the on hover here isn't really serving a purpose.